scripts/build-edit-substrate.sh

```
@@ -52,7 +52,12 @@ mkdir -p "$VENDOR_DIR"
   cat "$TMP_DIR/bundle.mjs"
 } > "$VENDOR_DIR/babel-parser.mjs"
 
-# Copy the MIT license file (required by the MIT license).
-cp node_modules/@babel/parser/LICENSE "$VENDOR_DIR/LICENSE"
+# Copy the MIT license text of every bundled package (required by the MIT license).
+: > "$VENDOR_DIR/LICENSE"
+for pkg in @babel/parser @babel/types @babel/helper-string-parser @babel/helper-validator-identifier; do
+  echo "===== $pkg =====" >> "$VENDOR_DIR/LICENSE"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)